src/utils.js

```diff
@@ -84,7 +84,8 @@ export function debounce (fn) {
 export function getClosestIgnoredElement (element) {
   var parent = element;
 
-  while (parent && parent !== document && !(parent instanceof DocumentFragment)) {
+  // e.g. document doesn't have a function hasAttribute; no need to go further up
+  while (parent && typeof parent.hasAttribute === 'function' && !(parent instanceof DocumentFragment)) {
```

Comment:
Also no one stops me from doing this ;)
![screen shot 2015-09-02 at 13 41 11](https://cloud.githubusercontent.com/assets/347650/9629020/5b8b084c-5178-11e5-8efd-204735415f09.png)